### PR TITLE
Math Magician Event

### PR DESCRIPTION
**Pull request by Bateyjosue**
# Feature Description 
In this project, I add the logic needed to make the Calculator component I developed in the previous project actually work.

## Cheers :tada: 

# Happy Review :clap: :clap:

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,44 +1,58 @@
 import React from 'react';
+// import Operator from './Operator';
+// import Operand from './Operand';
```
Comment (brytebee) [inline src/components/Calculator.js:3]: - [ ] **Optional:** Kindly remove commented-out and unused code to make your codebase cleaner and more professional.

File: src/logic/operate.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (brytebee) [inline src/logic/operate.js:1]: - [ ] **Optional:** Same as above

**Review by brytebee — APPROVED** [review]
Hi, @Bateyjosue👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Awesome UI design.✔️
- App functions as it should.✔️
- The readme is professional.✔️

##

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 

##

###  **Optional Suggestions:**
- N/A

------

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/logic/operate.js (hunk 2 above)
Comment (Bateyjosue) [reply src/logic/operate.js:1]: Thanks, bro for your comment and for your warm and friendly review. :pray: :pray:

**Pull request merged**